nmagent/client.go

```diff
@@ -284,6 +283,36 @@ func (c *Client) GetHomeAz(ctx context.Context) (AzResponse, error) {
 	return homeAzResponse, nil
 }
 
+func (c *Client) GetInterfaceIPInfo(ctx context.Context) (Interfaces, error) {
```

Comment:
It would be nice to put a doc comment here of the form: `// GetInterfaceIPInfo does x, y, z`